app/tests/src/test_chat_engine.py

```diff
@@ -1,24 +1,9 @@
 from src import chat_engine
-from src.chat_engine import GuruMultiprogramEngine, GuruSnapEngine
 
 
 def test_available_engines():
     engines = chat_engine.available_engines()
     assert isinstance(engines, list)
     assert len(engines) > 0
-    assert "guru-multiprogram" in engines
-    assert "guru-snap" in engines
-
-
-def test_create_engine_Guru_Multiprogram():
-    engine_id = "guru-multiprogram"
-    engine = chat_engine.create_engine(engine_id)
-    assert engine is not None
-    assert engine.name == GuruMultiprogramEngine.name
-
-
-def test_create_engine_Guru_SNAP():
-    engine_id = "guru-snap"
-    engine = chat_engine.create_engine(engine_id)
-    assert engine is not None
-    assert engine.name == GuruSnapEngine.name
```

Comment:
Add a check like this for one of the remaining engines

Reply:
Added tests for ImagineLA and CA EDD